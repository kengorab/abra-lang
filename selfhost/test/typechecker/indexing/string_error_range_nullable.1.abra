val str = "abcd"
val a: Int? = Option.Some(1)
str[a:2]