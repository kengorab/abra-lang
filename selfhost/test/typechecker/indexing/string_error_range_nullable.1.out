Error at %FILE_NAME%:3:5
Type mismatch
  |  str[a:2]
         ^
Expected: Int
but instead found: Int?
